automation.md

```diff
@@ -2,6 +2,10 @@
 
 Gravwell provides several utilities to enable automated operations. At the most basic level, users can schedule searches to be executed at specific times, for example every morning at 1:00. They can also schedule flows and scripts, which can execute multiple searches, sift and parse search results, and send notifications via email or HTTP. Flows are designed using a drag-and-drop graphical interface and are generally preferred over scripts, which should be reserved for legacy situations or very particular use cases that flows cannot cover.
 
+```{attention}
+Automations run as the user who owns the automation. This includes flows, scheduled searches, and scripts. When you share an automation with another user or group, you are granting them access to modify and execute the automation as your user. Consider using a machine user with the least privileges necessary as the owner of shared automations. 
+```
```

Comment:
Just did a quick check of our automations. The following support group-write sharing on 5.4.6:

- Alerts
- Flows
- Scheduled Search

The following **DO NOT** support group-write sharing yet:

- Scripts

I talked with @ashnwade a bit, and since this PR is targeting `master` (the 5.4.6 docs), we agreed that it made sense to modify the language here to be just a bit more precise. Scripts can't be group-write shared via the UI yet.


````suggestion
```{attention}
An automation (Flow, Scheduled Search, or Script) runs as the user who owns the automation. 

Granting write access to automations (currently Flows and Scheduled Searches) has important implications.

When you grant a group write access to a Flow or Scheduled Search, you are granting them the ability to modify and execute it as **your user**. Consider using a machine user with the least privileges necessary as the owner of shared automations. 
```
````

Please feel free to modify the above suggestion.